py3status/modules/mega.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""
+Display status of MEGAcmd.
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for the module (default "MEGA {format_sync}|No MEGA")
+    format_sync: display format for every sync (default "{syncstate}")
+    format_sync_separator: show separator if more than one sync (default " ")
+
+Format placeholders:
+    {format_sync} Format for every sync returned by 'mega-sync' command.
+
+format_sync placeholders:
+    Any column returned by 'mega-sync' command - in lower case!
+    For example: id, syncstate, localpath
+
+Requires:
+    MEGAcmd: command-line interface for MEGA
+
+@author Maxim Baz (https://github.com/maximbaz)
+@license BSD
+
+SAMPLE OUTPUT
+{'full_text': 'MEGA Synced'}
+"""
+
+STRING_NOT_INSTALLED = "MEGAcmd is not installed"
+
+
+class Py3status:
+    """
+    """
+
+    # available configuration parameters
+    cache_timeout = 10
+    format = "MEGA {format_sync}|No MEGA"
```

Comment:
Ha. Did you test this? I think this will come out as `'MEGA '`.

`'[MEGA {format_sync}|No MEGA]'` should be what makes the condition work.

Speaking of `'MEGA '`, I think `format_sync = ""` is a bad approach because it will (generally) hide things
from users like this example I'd rather we either pass `None` or we don't pass anything at all to the dict... so we see stray `{format_sync}` instead.

* `'MEGA '`
* `'MEGA None'`
* `'MEGA {format_sync}'`  <--- my preference, more clean.

Reply:
I did test, it works as expected. Changed to None instead of empty string, behaves in the same way, but if you prefer it this way, let it be so.

Reply:
>I did test, it works as expected.

It did not work before (twas tested), but now it works... because of recent revert 5fdb2b3d.